xcodeproj/internal/providers.bzl

```diff
@@ -67,7 +67,6 @@ target type are allowed to propagate.
 
 target_type = struct(
     compile = "compile",
-    resources = "resources",
 )
```

Comment:
This whole struct will be removed after PR, as this reworking unblocks further reworking of input file and transitive dependency processing.